index.html

```diff
@@ -9222,6 +9225,256 @@ <h3 id="example-2">
+          </p>
+          <ul>
+            <li>Fixed look-ahead (this means that an
+            <a>DynamicsCompressorNode</a> adds a fixed latency to the signal
+            chain).
+            </li>
+            <li>Configurable attack speed, release speed, threshold, knee
+            hardness and ratio.
+            </li>
+            <li>Side-chaining is not supported
+            </li>
+            <li>The gain reduction is reported <em>via</em> the
+            <code>reduction</code> property on the
+            <a>DynamicsCompressorNode</a>
+            </li>
+            <li>The compression curve is has three parts:
+              <ul>
+                <li>The first part is the identify: \(f(x) = x\).
+                </li>
+                <li>The second part is the soft-knee portion, which MUST be a
+                monotonically increasing function.
+                </li>
+                <li>The third part is a linear function of equation \(f(x) =
+                \frac{1}{ratio} \cdot x \).
+                </li>
+              </ul>This curve MUST be continuously differentiable, and
+              corresponds to a target output level, based on the input level.
+            </li>
+          </ul>
+          <p>
+            Graphically, such a curve would look something like this:
+          </p>
+          <figure>
+            <img alt="Graphical representation of a compression curve" src=
+            "images/compression-curve.svg" style="width: 50%">
+            <figcaption>
+              A typical compression curve, showing the knee portion (soft or
+              hard) as well as the threshold.
+            </figcaption>
+          </figure>
+          <p>
+            Internally, the <a>DynamicsCompressorNode</a> is described with a
+            combination of other <a>AudioNode</a>s, as well as a special
+            algorithm, to compute the gain reduction value.
+          </p>
+          <p>
+            The following <a>AudioNode</a> graph is used internally,
+            <code>input</code> and <code>output</code> respectively being the
+            input and output <a>AudioNode</a>, <code>context</code> the
+            <a>BaseAudioContext</a> for this <a>DynamicsCompressorNode</a>, and
+            new class, <code>EnvelopeFollower</code>, that instantiate a
+            special object that behaves like an <a>AudioNode</a>, described
+            below:
+          </p>
+          <pre>
+              var delay = new DelayNode(context, {delayTime: 0.006});
+              var gain = new GainNode(context);
+              var compression = new EnvelopeFollower();
+
+              input.connect(delay).connect(gain).connect(output);
+              input.connect(compression).connect(gain.gain);</pre>
+          <figure>
+            <img src="images/dynamicscompressor-internal-graph.svg" alt=
+            "Schema of
+            the internal graph used by the DynamicCompressorNode">
+            <figcaption>
```

Comment:
Thanks for the diagram!  Looks really nice.  One nit:  Kind of hard to tell that the envelope follower output connects to the gain attribute of the gain node.  Not exactly sure how to best illustrate this, though.